src/renderer/extensions/vueNodes/components/NodeHeader.vue

```diff
@@ -32,17 +32,39 @@
         @cancel="handleTitleCancel"
       />
     </div>
+
+    <!-- Title Buttons -->
+    <div v-if="!readonly" class="flex items-center">
+      <IconButton
+        v-if="isSubgraphNode"
+        size="sm"
+        type="transparent"
+        class="text-stone-200 dark-theme:text-slate-300"
+        data-testid="subgraph-enter-button"
+        title="Enter Subgraph"
+        :on-click="handleEnterSubgraph"
```

Comment:
Or is this a special handler?

```suggestion
        @click="handleEnterSubgraph"
```